src/storage.rs

```diff
@@ -37,6 +37,7 @@ fn create_main_tables_if_needed(conn: &DatabaseConnection) {
         "CREATE TABLE IF NOT EXISTS {} (
         id TEXT PRIMARY KEY,
         name TEXT,
+        pinned_message_id INTEGER,
```

Comment:
Probably should make this an array now if we want to have multiple pinned messages in future without another migration